rplugin/python3/deoplete/sources/beancount.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, vim):
         self.name = 'beancount'
         self.mark = '[bc]'
         self.filetypes = ['beancount']
+        self.rank = 500
```

Comment:
It makes this source always outrank the basic ones (like the buffer completion), so there's no more need to disable them.